clients/admin-ui/src/features/datastore-connections/edit-connection/EditConnection.tsx

```diff
@@ -0,0 +1,86 @@
+import { Box, Flex, Heading, Text } from "@fidesui/react";
+import React, { useEffect, useState } from "react";
+
+import { useAppDispatch, useAppSelector } from "~/app/hooks";
+import { selectConnectionTypeState, setStep } from "~/features/connection-type";
+
+import Breadcrumb from "../add-connection/Breadcrumb";
+import ConfigurationSettingsNav from "../add-connection/ConfigurationSettingsNav";
+import { ConnectorParameters } from "../add-connection/ConnectorParameters";
+import {
+  ConfigurationSettings,
+  CONNECTOR_PARAMETERS_OPTIONS,
+  STEPS,
+} from "../add-connection/constants";
+import DatasetConfiguration from "../add-connection/DatasetConfiguration";
+import DSRCustomization from "../add-connection/manual/DSRCustomization";
+import { ConnectorParameterOption } from "../add-connection/types";
+import ConnectionTypeLogo from "../ConnectionTypeLogo";
+
+const EditConnection: React.FC = () => {
```

Comment:
~~Threading here because I can't spot where the problem is: I'm not able to update most of the fields of a database (postgres) connection. When I hit save, the PATCH payload doesn't have most of the fields.~~

Ohh, [your comment](https://github.com/ethyca/fides/issues/1204#issuecomment-1273603449) helped me understand that the secrets are stored separately, sent through a different request, and aren't re-loaded in the form (for security?). This was definitely confusing UX -- at very least it seems like the form should populate with the non-password fields.

Reply:
Your correct the secrets are not reloaded on the form due to security reasons. There are UI/UX discussions to mask the secret values initially when the form is rendered. The user can reveal the secret values one input at a time or toggle them all at once. Pending future enhancement in an upcoming sprint.